### ISSUE TITLE
Debug not printing all files!

### ISSUE THREAD

**Issue by DaveyD** [sindresorhus/gulp-debug#34]
Hi, I was using debug to check my glob pattern and realized something's not working correctly.
I have a file tree as follows: (caps are folders)
-MAIN
---style1.css
---DEV
------style2.css
------style3.css
------style4.css
---DIST
------style2.min.css
------style3.min.css
------style4.min.css

I used the following glob pattern: `MAIN/**/*.css` and only the files from the DEV folder got printed
I tested the same glob other ways and they all reported all files.
I tested it using the original glob plugin, I also tested it through gulp using a separate callback function - they both printed the files from both directories as expected.

Is this a bug?
Thanks,
David

**Comment by sindresorhus** [comment]
This plugin just prints what's in the stream. You probably didn't match the glob correctly.

I would suggest asking on Stack Overflow: http://stackoverflow.com/questions/tagged/gulp

**Comment by DaveyD** [comment]
Thanks, but I'm not sure you read all that I wrote
I also expected that to be true, so at first I suspected that my glob pattern wasn't correct. But after testing it with some other plugins verifying that it was correct, I came to the conclusion that it must be something with the gulp-debug plugin.
Take this for example:
````
gulp.task( 'test_glob', function() {
    gulp.src(base + '**/*.css')
        .pipe(debug())
        .pipe(gulpfn(function(file) {
            console.log("myDebug: " + file.path);
        }));
});
````

When I run this task as is, all files get printed to the console as expected. However, if I remove the second pipe and leave only the pipe to debug, then only files from the DEV folder get printed.
Really weird...!
